src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -35,6 +35,7 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Average = _library.Append(new AverageFunction());
         public static readonly TexlFunction AverageT = _library.Append(new AverageTableFunction());
         public static readonly TexlFunction Blank = _library.Append(new BlankFunction());
+        public static readonly TexlFunction Boolean = _library.Append(new BooleanFunction());
```

Comment:
> Boolean = _library.Append(new BooleanFunction()); [](http://example.com/codeflow?start=43&length=50)

This will require a corresponding change to PAClient.